x-pack/plugins/apm/public/components/shared/useDelayedVisibility/index.test.tsx

```diff
@@ -58,28 +58,29 @@ describe.skip('useFetcher', () => {
     expect(hook.result.current).toEqual(true);
   });
 
-  it('is true for minimum 1000ms', () => {
```

Comment:
You can use [`it.skip`](https://jestjs.io/docs/en/api.html#testskipname-fn) here.

Reply:
True, that should have been better